lib/kafo/puppet_command.rb

```diff
@@ -50,11 +50,11 @@ def self.aio_puppet?
       false
     end
 
-    def self.format_command(command)
+    def self.format_command(command, extra_env = {})
       if aio_puppet?
```

Comment:
Perhaps refactor this method to
```ruby
env = aio_puppet? clean_env_vars : ::ENV.to_h
[env.merge(extra_env), command, :unsetenv_others => false]
```

Reply:
Considered, but `unsetenv_others` is also different in those branches.

Reply:
Ah yes, now I see why I wrote it that way in the past.